data/decks/web.yaml

```diff
@@ -1,6 +1,12 @@
 ---
-name: Web test deck
-description: This deck runs HTTP Header Field Manipulation and the Web Connectivity test
+name: Test URLs
+description: "This deck examines whether (and how) websites are blocked and
+tries to detect censorship and/or surveillance systems in your network. The Web
+Connectivity test examines whether access to websites is blocked through DNS
+tampering, TCP/IP blocking, or by a transparent HTTP proxy. The
+HTTP Header Field Manipulation test tries to detect censorship and/or
```

Comment:
As the category name changed to 'Test URLs'  perhaps makes more sense to add the HTTP Header Field Manipulation test to 'Test for Middle Boxes' but in a different deck than http-invalid.yaml --so that we can still disable it separately.